app/mailers/faculty_notifications_mailer.rb

```diff
@@ -19,4 +19,13 @@ def scholarsphere_deposit_failure(user, deposit)
          from: 'scholarsphere@psu.edu',
          reply_to: 'scholarsphere@psu.edu'
   end
+
+  def wrong_file_version(publications)
+    @publications = publications
+    #will need to change the mail to to whatever connects a publication to the activity insight uploader's email
```

Comment:
This is now an attribute of the file record. So `ActivityInsightOAFile#user` is the RMD user who uploaded the file in Activity Insight. This change has been merged into main.